modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/display/view.jsp

```diff
@@ -67,9 +67,7 @@ boolean limitToOneSubmissionPerUser = DDMFormInstanceSubmissionLimitStatusUtil.i
 
 		boolean expired = false;
 
-		if (ddmFormDisplayContext.isExpirationDateEnabled()) {
-			expired = DDMFormInstanceExpirationStatusUtil.isFormExpired(formInstance, timeZone);
-		}
+		expired = DDMFormInstanceExpirationStatusUtil.isFormExpired(formInstance, timeZone);
```

Comment:
You can declare and assign in the same line like so:
```
boolean expired = DDMFormInstanceExpirationStatusUtil.isFormExpired(formInstance, timeZone);
boolean preview = ddmFormDisplayContext.isPreview();
boolean showSuccessPage = ddmFormDisplayContext.isShowSuccessPage();
```

Reply:
Thanks Paulino, I will do that!